pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/PlayerView.kt

```diff
@@ -156,19 +154,88 @@ fun PlayerView(
             )
         }
 
-        BlockedTimeRangeWarning(
```

Comment:
We don't display block messages anymore?

Reply:
Yes to be aligned with apple demo :D